rs/p2p/quic_transport/README.adoc

```diff
@@ -65,13 +62,34 @@ In order to make request processing https://docs.rs/tokio/latest/tokio/task/inde
 Job execution starts in the threadpool iff the parent task was not cancelled.
 A nice writeup about async and blocking operations can be found in https://ryhl.io/blog/async-what-is-blocking/[Alice Ryhl's blog post].
 
-== Implementation design decisions ==
+== Implementation Details ==
+
+Use the https://github.com/quinn-rs/quinn[QUINN] library for QUIC, as it meets the requirements for reliable data delivery, multiplexing, connection-oriented communication, and flow control.  
+
+=== QUIC stream per message ===
+
+Instead of one stream per handler/URI. TODO: explain.
+
+=== Handlers are always ready to process messages ===
+
+The router is always ready to accept streams. TODO: explain.
```

Comment:
Would be good to create a Jira ticket for these.

Reply:
https://dfinity.atlassian.net/browse/NET-1779